tests/memgraph/test_query_builder.py

```diff
@@ -283,14 +283,14 @@ def test_load_csv_no_header(memgraph):
     mock.assert_called_with(expected_query)
 
 
-def test_where(memgraph):
+def test_where_literal(memgraph):
     query_builder = (
         QueryBuilder()
         .match()
         .node("L1", variable="n")
         .to("TO")
         .node("L2", variable="m")
-        .where("n.name", "=", "best_name")
+        .where("n.name", "=", literal="best_name")
```

Comment:
Can we use keyword arguments for all of the parameters?

Reply:
We can. They are currently not used because we know that on the left-hand side, there must be an item that is a property or variable and that in the middle must be an operator, and that there is no other option. Keyword argument was added just for the right-hand side because it's important to emphasize whether it's a literal or an expression. I would like keyword arguments for parameters better, so we can further discuss this tomorrow on Research Wednesday.